config/application.rb

```diff
@@ -38,6 +38,7 @@ class Application < Rails::Application
     config.i18n.enforce_available_locales = true
 
     config.active_record.observers = :liquid_partial_observer
+    config.assets.paths += (Settings.custom_assets_paths || "").split(":") if Settings.custom_assets_paths.present?
 
```

Comment:
Nice PR. Perhaps a little zealous with the nil checking? What about ditching the conditional and using `try` or `to_s`.

`Settings.custom_assets_paths.to_s.split(':')` or
`Settings.customer_assets_paths.try(:split, ':')`